src/main/webapp/js/components/dev/ExperimentsTable/ExperimentsTable.js

```diff
@@ -0,0 +1,524 @@
+			} else if (experiment.getStatus() == GEPPETTO.Resources.ExperimentStatus.ERROR)
+			{
+				tdStatus =<td id="statusIcon"><div className="circle ERROR center-block" title="ERROR"></div></td>;
+			}
+
+			return (tdStatus);
+		}
+	});
+	
+	/**
+	 * Creates html element holding set of icons used for experiment controls within table
+	 */
+	var IconsElement = React.createClass({
+		getInitialState: function() {
+			return {visible: false};
+		},
+		show : function(){
+	        this.setState({ visible: true });
+		},
+		hide : function(){
+	        this.setState({ visible: false });
+		},
+		componentDidMount : function(){
+			var experiment = this.props.experiment;
+			
+			var activeIconId = "#activeIcon-" +experiment.getId();
+			var downloadResultsIconId = "#downloadResultsIcon-" +experiment.getId();
+			var downloadModelsIconId = "#downloadModelsIcon-"+ experiment.getId();
+			var deleteIconId = "#deleteIcon-" +experiment.getId();
+			
+			$(activeIconId).click(function(e) {
+				 var index = window.Project.getExperiments().indexOf(experiment);
+		         GEPPETTO.trigger('show_spinner', GEPPETTO.Resources.LOADING_EXPERIMENT);
+		         GEPPETTO.Console.executeCommand("Project.getExperiments()[" + index + "].setActive();");
+		         e.stopPropagation();
+			})
+			$(deleteIconId).click(function(e) {
+				var index = window.Project.getExperiments().indexOf(experiment);
+	            GEPPETTO.Console.executeCommand("Project.getExperiments()[" + index + "].deleteExperiment();");
+				e.stopPropagation();
+			})
+			$(downloadModelsIconId).click(function(e) {
+				var simulatorConfigurations = experiment.simulatorConfigurations;
+	            for (var config in simulatorConfigurations) {
+	                var simulatorConfig = simulatorConfigurations[config];
+	                GEPPETTO.Console.executeCommand('Project.downloadModel("' + simulatorConfig["aspectInstancePath"] + '");');
+	            }
+				e.stopPropagation();
+			})
+			$(downloadResultsIconId).click(function(e) {
+				 var index = window.Project.getExperiments().indexOf(experiment);
+	             var simulatorConfigurations = experiment.simulatorConfigurations;
+	             for (var config in simulatorConfigurations) {
+	                 var simulatorConfig = simulatorConfigurations[config];
+	                 GEPPETTO.Console.executeCommand("Project.getExperiments()[" + index + "].downloadResults('" + simulatorConfig["aspectInstancePath"] + "'," + "'RAW');");
+	             }
+				e.stopPropagation();
+			})
+		},
+		render: function() {
+			var experiment = this.props.experiment;
+			//Create IDs for icons
+			var activeIconId = "activeIcon-" +this.props.experiment.getId();
+			var deleteIconId = "deleteIcon-" +this.props.experiment.getId();
+			var downloadResultsIconId = "downloadResultsIcon-" +this.props.experiment.getId();
+			var downloadModelsIconId = "downloadModelsIcon-" +this.props.experiment.getId();
+    
+			var style  = {
+					paddingRight:"10px",
```

Comment:
Curious about this extra style information -- does it not work to have this in the less file?